src/spec/solver_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec'
+require_relative '../solver_class'
+
+describe Solver do
+  before(:each) do
+    @solver = Solver.new
+  end
+
+  context '#factorial' do
+    it 'returns 720 for input 6' do
+      expect(@solver.factorial(6)).to eq 720
+    end
+  end
+
+  context '#reverse' do
+    it 'returns "olleh" for input "hello"' do
+      expect(@solver.reverse('hello')).to eq 'olleh'
+    end
+  end
+
+  context '#fizzbuzz' do
+    it 'returns "1 2 fizz 4 buzz fizz 7 8 fizz buzz 11 fizz 13 14 fizzbuzz" for the first 15 numbers' do
+      expect(@solver.fizzbuzz(15)).to eq '1 2 fizz 4 buzz fizz 7 8 fizz buzz 11 fizz 13 14 fizzbuzz'
+    end
+  end
```

Comment:
- [ ] It would be a good thing to break this down and have at least 4 test cases as provided in the project requirements. eg ```expect(solver.fizzbuzz(3)).to eql('fizz')```

![image](https://user-images.githubusercontent.com/86318284/211061897-70881425-dcfc-4387-96cc-93a49b0ae7fb.png)

Reply:
Done!